v5/core/common_test.go

```diff
@@ -39,42 +41,45 @@ func toJSON(obj interface{}) string {
-	"SERVICE3_DISABLE_SSL":                   "false",
-	"SERVICE3_ENABLE_GZIP":                   "notabool",
-	"SERVICE3_AUTH_TYPE":                     "Cp4D",
-	"SERVICE3_AUTH_URL":                      "https://cp4dhost/cp4d/api",
-	"SERVICE3_USERNAME":                      "my-cp4d-user",
-	"SERVICE3_PASSWORD":                      "my-cp4d-password",
-	"SERVICE3_AUTH_DISABLE_SSL":              "false",
-	"EQUAL_SERVICE_URL":                      "https://my=host.com/my=service/api",
-	"EQUAL_SERVICE_APIKEY":                   "===my=iam=apikey===",
-	"SERVICE6_AUTH_TYPE":                     "iam",
-	"SERVICE6_APIKEY":                        "my-api-key",
-	"SERVICE6_SCOPE":                         "A B C D",
-	"SERVICE7_AUTH_TYPE":                     "container",
-	"SERVICE7_CR_TOKEN_FILENAME":             "crtoken.txt",
-	"SERVICE7_IAM_PROFILE_NAME":              "iam-user2",
-	"SERVICE7_IAM_PROFILE_ID":                "iam-id2",
-	"SERVICE7_AUTH_URL":                      "https://iamhost/iam/api",
-	"SERVICE7_CLIENT_ID":                     "iam-client2",
-	"SERVICE7_CLIENT_SECRET":                 "iam-secret2",
-	"SERVICE7_SCOPE":                         "scope2 scope3",
+	"SERVICE_1_URL":              "https://service1/api",
+	"SERVICE_1_DISABLE_SSL":      "true",
+	"SERVICE_1_ENABLE_GZIP":      "true",
+	"SERVICE_1_AUTH_TYPE":        "IaM",
+	"SERVICE_1_APIKEY":           "my-api-key",
+	"SERVICE_1_CLIENT_ID":        "my-client-id",
+	"SERVICE_1_CLIENT_SECRET":    "my-client-secret",
+	"SERVICE_1_AUTH_URL":         "https://iamhost/iam/api",
+	"SERVICE_1_AUTH_DISABLE_SSL": "true",
+	"SERVICE2_URL":               "https://service2/api",
+	"SERVICE2_DISABLE_SSL":       "false",
+	"SERVICE2_ENABLE_GZIP":       "false",
+	"SERVICE2_AUTH_TYPE":         "bAsIC",
+	"SERVICE2_USERNAME":          "my-user",
+	"SERVICE2_PASSWORD":          "my-password",
+	"SERVICE3_URL":               "https://service3/api",
+	"SERVICE3_DISABLE_SSL":       "false",
+	"SERVICE3_ENABLE_GZIP":       "notabool",
+	"SERVICE3_AUTH_TYPE":         "Cp4D",
+	"SERVICE3_AUTH_URL":          "https://cp4dhost/cp4d/api",
+	"SERVICE3_USERNAME":          "my-cp4d-user",
+	"SERVICE3_PASSWORD":          "my-cp4d-password",
+	"SERVICE3_AUTH_DISABLE_SSL":  "false",
+	"EQUAL_SERVICE_URL":          "https://my=host.com/my=service/api",
+	"EQUAL_SERVICE_APIKEY":       "===my=iam=apikey===",
+	"SERVICE6_AUTH_TYPE":         "iam",
+	"SERVICE6_APIKEY":            "my-api-key",
+	"SERVICE6_SCOPE":             "A B C D",
+	"SERVICE7_AUTH_TYPE":         "container",
+	"SERVICE7_CR_TOKEN_FILENAME": "crtoken.txt",
+	"SERVICE7_IAM_PROFILE_NAME":  "iam-user2",
+	"SERVICE7_IAM_PROFILE_ID":    "iam-id2",
+	"SERVICE7_AUTH_URL":          "https://iamhost/iam/api",
+	"SERVICE7_CLIENT_ID":         "iam-client2",
+	"SERVICE7_CLIENT_SECRET":     "iam-secret2",
+	"SERVICE7_SCOPE":             "scope2 scope3",
+	"SERVICE8_AUTH_TYPE":         "VPC",
```

Comment:
Only added the entries for service8.  The rest of the changes are just due to formatting differences.